functions.php

```diff
@@ -231,4 +231,105 @@ function sc_add_ellipsis_to_behind_the_style_posts( $excerpt ) {
 	}
 	return $excerpt;
 }
-add_filter( 'get_the_excerpt', __NAMESPACE__ . '\sc_add_ellipsis_to_behind_the_style_posts' );
```

Comment:
Let's not use magic variables for call backs. Can we stick to array(classname  or $this, 'sc_add_ellipsis_to_behind_the_style_posts') -- just a lot easier to read in isolation.